h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -1282,7 +1320,7 @@ else if ((long)min == min)
     if( leRange < 255 ) {    // Span fits in a byte?
       if(0 <= min && max < 255 ) // Span fits in an unbiased byte?
         return new C1Chunk( bufX(0,0,C1Chunk._OFF,0));
-      return new C1SChunk( bufX(lemin,xmin,C1SChunk._OFF,0),lemin,xmin);
```

Comment:
Can you please explain?

Reply:
Michal:  For binary columns, xmin is 0 always, min_l == lemin ==0

Reply:
Since it did not add to anything, I will change it back to lemin.